setup.cfg

```diff
@@ -31,6 +31,7 @@ classifiers =
 [options]
 python_requires = >=3.7
 install_requires =
+    asgiref >= 3.4.1
```

Comment:
Would `asgiref >= 3.3.2` be fine as well or do we need the newer version? Django 3.2.x specifies `asgiref >= 3.3.2 <4` in `setup.cfg` and I'd prefer to reuse this.

Reply:
Either should work fine I think - I just had to pick one and went with what the latest Django used.  Updated to the same as Django 3.2.x.